nodemon.json

```diff
@@ -1,11 +1,7 @@
 {
   "watch": [
-    "controllers",
-    "middleware",
-    "models",
-    "routes",
-    "utilities"
+    "server"
   ],
   "ext": "js",
-  "exec": "node --inspect=0.0.0.0:9229 app.js"
+  "exec": "NODE_ENV=development node --inspect=0.0.0.0:9229 server/app.js"
```

Comment:
this might break takikkos setup, NODE_ENV dev should be filled automatically on startup by dotenv